pkgs/development/libraries/libuv/default.nix

```
@@ -71,7 +71,7 @@ stdenv.mkDerivation rec {
     homepage    = "https://libuv.org/";
     changelog   = "https://github.com/libuv/libuv/blob/v${version}/ChangeLog";
     maintainers = with maintainers; [ cstrahan ];
-    platforms   = with platforms; linux ++ darwin;
+    platforms   = with platforms; linux ++ darwin ++ windows;
```

I think we can use
```suggestion
    platforms   = platforms.all;
```
besides that, LGTM